### PR TITLE
Feature display user profile kb 008

### PR DESCRIPTION
**Pull request by alvase**

### CODE REVIEW

File: src/Components/MainPageComponent/ArticlePreview.js [hunk 1]
```diff
@@ -31,50 +35,58 @@ const InternalArticlePreview = props => {
...
-              </a>
-              <div className='info'>
-                <a href='#top' className='author'>
-                  {article.author.username}
-                </a>
-                <span className='date'>
-                  {dateFormat(article.updatedAt, "ddd mmm dd yyyy")}
-                </span>
+
+      {/* 文章 */}
```
Comment (alvase) [inline src/Components/MainPageComponent/ArticlePreview.js]: Use English if possible in document and comment.

File: src/ReduxStore/FeedDetails/feedReducers.js [hunk 2]
```diff
@@ -41,12 +47,20 @@ export const feedReducer = (state = initialState, action) => {
...
       return { ...state, tagRelatedArticles: action.tagRelatedArticles };
 
-    case POPULAR_TAG_DISPLAYED:
-      return { ...state, isDisplay: action.tagName };
-
     case RELATED_TAG_LOADED:
       return { ...state, currentTagName: action.tagName };
 
+    case GLOBLE_FEED_CLICKED:
+      return { ...state, currentTagName: "", tagRelatedArticles: [] };
```
Comment (alvase) [inline src/ReduxStore/FeedDetails/feedReducers.js]: Set `tagRelatedArticles` to empty array will cause problem when clicking global feed link

File: src/ReduxStore/FeedDetails/feedReducers.js [hunk 3]
```diff
@@ -15,7 +18,10 @@ const initialState = {
   articleLibrary: [],
   popularTags: [],
   currentArticleTitle: "",
-  currentArticleSlug: ""
+  currentArticleSlug: "",
+  currentProfileData: {},
+  currentUsersArticles: [],
+  favoritedArticles:[]
 };
```
Comment (alvase) [inline src/ReduxStore/FeedDetails/feedReducers.js]: Better to have `tagRelatedArticles` into the initialState.

File: src/Components/UserComponent/UserProfile.js [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+                    Favorited Articles
+                  </a>
+                </li>
+
+
+              </ul>
+            </div>
+
+            {/* 一篇文章 */}
+            {props.currentUsersArticles.map((article,index) => {
```
Comment (alvase) [inline src/Components/UserComponent/UserProfile.js]: `favoritedArticles` is connected but not used here.

File: src/ReduxStore/FeedDetails/feedReducers.js (hunk 2 above)
Comment (StevenWang9102) [reply src/ReduxStore/FeedDetails/feedReducers.js]: fixed

File: src/ReduxStore/FeedDetails/feedReducers.js (hunk 3 above)
Comment (StevenWang9102) [reply src/ReduxStore/FeedDetails/feedReducers.js]: fixed

File: src/Components/UserComponent/UserProfile.js (hunk 4 above)
Comment (StevenWang9102) [reply src/Components/UserComponent/UserProfile.js]: fixed

**Pull request merged**